cmd/cadvisor.go

```diff
@@ -124,6 +149,31 @@ func main() {
 	klog.V(1).Infof("enabled metrics: %s", includedMetrics.String())
 	setMaxProcs()
 
+	plugins := map[string]container.Plugin{
+		"containerd": containerd.NewPluginWithOptions(&containerd.Options{ // TODO(@spartan0x117): Make containerdEnvMetadataWhiteList non-global and add to these options
```

Comment:
`envMetadataWhiteList` is actually already [non-global and is passed](https://github.com/google/cadvisor/blob/master/cmd/cadvisor.go#L136C203-L136C203) into `manager.New()`
```suggestion
		"containerd": containerd.NewPluginWithOptions(&containerd.Options{  options
```